packages/portal-proto/src/features/genomic/filters.json

```diff
@@ -14,14 +14,13 @@
   {
     "facet_filter": "genes.biotype",
     "name": "Biotype",
-    "description": "No description",
+    "description": null,
```

Comment:
this was to get around: 
> src/features/genomic/FilterPanel.tsx(93,30): error TS2339: Property 'description' does not exist on type '{ facet_filter: string; name: string; type: string; docType: string; }'.

please let me know if there is a better way.

Reply:
per pari's suggestion, I ended up removing `description` from the cards since we don't have any descriptions. Figured we can add it back in if we ever have descriptions.